src/ssb_project_cli/ssb_project/app.py

```diff
@@ -560,8 +595,8 @@ def get_kernels_dict() -> dict[str, str]:
 
 def main() -> None:
     """Main function of ssb_project_cli."""
-    app(prog_name="ssb-project")
```

Comment:
Do we need? # pragma: no cover

Reply:
Unless we plan to test it somehow, coverage says the line is not covered, but no big deal